source/making_payments/index.html.md.erb

```diff
@@ -153,7 +153,7 @@ progress. Alternatively, a service can make the following API call for a given
 A `204` response indicates success. Any other response indicates an error.
 
 See <a href="https://govukpay-api-browser.cloudapps.digital/#cancelpayment"
-target="blank">API browser</a> [external link] for more details.
```

Comment:
Could say `See the <a href="https://govukpay-api-browser.cloudapps.digital/#cancelpayment" target="blank">GOV.UK Pay API browser</a> for more details`

Reply:
Thanks Jon - fixed!